qml/components/chatInformationPage/ChatInformationTabItemBase.qml

```diff
@@ -30,15 +30,12 @@ Item {
     property alias loadingVisible: loadingColumn.loadingVisible
     property string loadingText
 
-    property int tabIndex: tabItem.VisualItemModel.index
-    property bool active: tabItem.ListView.isCurrentItem
+    property int tabIndex: index
+    property bool active: index === tabView.currentIndex
 
     default property alias _data: contentItem.data
 
-
-    width: parent.width
-    height: tabView.maxHeight//Math.max(contentItem.height, loadingColumn.height)
-    opacity: active ? 1.0 : 0.0
+    opacity: active ? 1.0 : 0.2
     Behavior on opacity { PropertyAnimation {duration: 300}}
```

Comment:
Better to use `FadeAnimation` for all opacity behaviors for visual consistency. Unless this one is made 100ms longer than `FadeAnimation` on purpose (which I doubt)

Reply:
I actually like 300ms, but you're right. 
Also I missed the 0.2 above which I didn't want to commit.

Reply:
It looks disturbing when different things on the same screen are being animated slightly out of sync))